web/tomato/js/jquery.contextMenu.js

```diff
@@ -204,6 +204,10 @@ var // currently active contextMenu trigger
     handle = {
         // abort anything
         abortevent: function(e){
+        	var $this = $(this);
```

Comment:
I wanted to use the same code style as the original author. It is used in line 208 to check for the correct style class as it references the parts of our context menus which should be able to allow copy-past with standard context menus.